Sources/Twift+Lists.swift

```diff
@@ -103,6 +103,35 @@ extension Twift {
   public func deleteList(_ listId: List.ID) async throws -> TwitterAPIData<DeleteResponse> {
     return try await call(route: .list(listId), method: .DELETE, expectedReturnType: TwitterAPIData.self)
   }
+    
+    /// Enables the authenticated user to create a new List.
```

Comment:
Nit: could you add a docblock comment indicating which API request this is equivalent to? See some of the other methods for an example.

```
  /// Enables the authenticated user to create a new List.
  ///
  /// Equivalent to `POST /2/lists`
```